src/domain/community/community/community.service.ts

```diff
@@ -155,6 +155,20 @@ export class CommunityService {
     return true;
   }
 
+  async getParentCommunity(
+    community: ICommunity
+  ): Promise<ICommunity | undefined> {
+    const communityWithParent = await this.getCommunityOrFail(community.id, {
+      relations: ['parentCommunity'],
```

Comment:
I think it's recursive structure and has a relation to community, parentCommunityId is the id of related community, but it's not the relation name, if I understand correctly